client/src/views/users/UserDetailsModal.vue

```diff
@@ -0,0 +1,206 @@
+<!-- Parsec Cloud (https://parsec.cloud) Copyright (c) BUSL-1.1 2016-present Scille SAS -->
+
+<template>
+  <ion-page class="modal">
+    <ms-modal
+      :title="$t('UsersPage.UserDetailsModal.title')"
+      :close-button="{ visible: true }"
+    >
+      <div>
+        <ion-text class="text1">
```

Comment:
We use BEM (Blocks Elements Modifiers) as a naming referecence.
Here's what it is exactly: https://alticreation.com/bem-pour-le-css/
I've made a code suggestion to see what it might looks like